app/scripts/migrations/002.js

```diff
@@ -1,12 +1,11 @@
-const version = 2
-
 import { cloneDeep } from 'lodash'
 
+const version = 2
 
 export default {
   version,
 
-  migrate: function (originalVersionedData) {
+  migrate (originalVersionedData) {
```

Comment:
What syntax is this O_O

Reply:
[Method definitions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions):

> Starting with ECMAScript 2015, a shorter syntax for method definitions on objects initializers is introduced. It is a shorthand for a function assigned to the method's name.